.github/workflows/ci.yml

```
@@ -63,7 +63,15 @@ jobs:
           override: true
 
       - name: Install scopelint
-        run: cargo install scopelint
+        uses: engineerd/configurator@v0.0.8
+        with:
+          name: scopelint
+          repo: ScopeLift/scopelint
+          fromGitHubReleases: true
+          version: latest
+          pathInArchive: scopelint-x86_64-linux/scopelint
+          urlTemplate: https://github.com/ScopeLift/scopelint/releases/download/{{version}}/scopelint-x86_64-linux.tar.xz
+          token: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Check formatting
         run: scopelint check
```

Just as a sanity check on the version being used in case there's any issues
```suggestion
        run: |
          scopelint --version
          scopelint check
```

---

added in 29b3bedc50644c79b9f947fe63322c7a78faefd3